modules/installing-ocp-agent-ibm-z.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/installing-with-agent-based-installer.adoc
+// * installing/installing_with_agent_based_installer/prepare-pxe-infra-agent.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-ocp-agent-ibm-z_{context}"]
+= {ibm-z-title} only: Manually adding {ibm-z-title} agents
+
+After creating the PXE assets you can add {ibm-z-name} agents. Depending on your {ibm-z-name} environment you can choose from the following:
+
+* Adding {ibm-z-name} agents with z/VM
+* Adding {ibm-z-name} agents with {op-system-base} KVM
+
+.Procedure
+
+** Adding {ibm-z-name} agents with z/VM
+
+. Create a parameter file for the z/VM guest.
++
+.Example parameter file
++
+[source,terminal]
+----
+rd.neednet=1 \
+console=ttysclp0 \
+coreos.live.rootfs_url=<rootfs_url> \ <1>
+ip=172.18.78.2::172.18.78.1:255.255.255.0:::none nameserver=172.18.78.1 \ <2>
+zfcp.allow_lun_scan=0 \ <3>
+rd.znet=qeth,0.0.bdd0,0.0.bdd1,0.0.bdd2,layer2=1 \
+rd.dasd=0.0.4411 \ <4>
+rd.zfcp=0.0.8001,0x50050763040051e3,0x4000406300000000 \ <5>
+random.trust_cpu=on rd.luks.options=discard \
+ignition.firstboot ignition.platform.id=metal \
+console=tty1 console=ttyS1,115200n8 \
+coreos.inst.persistent-kargs="console=tty1 console=ttyS1,115200n8"
+----
+<1> For the `coreos.live.rootfs_url=` artifact, specify the matching `rootfs` artifact for the `kernel`and `initramfs` you are booting. Only HTTP and HTTPS protocols are supported.
+<2> For the `ip=` parameter, assign the IP address automatically using DHCP or manually as described in "Installing a cluster with z/VM on {ibm-z-name} and {ibm-linuxone-name}".
```

Comment:
This might need revised for clarity. I'll take a stab at it: 

```suggestion
<2> For the `ip=` parameter, assign the IP address automatically using DHCP, or manually assign the IP address, as described in "Installing a cluster with z/VM on {ibm-z-name} and {ibm-linuxone-name}".
```